core/src/main/kotlin/com/exactpro/th2/read/db/impl/grpc/ExecuteBodyData.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.th2.read.db.impl.grpc
+
+import com.exactpro.th2.common.event.IBodyData
+import com.exactpro.th2.read.db.core.DataSourceConfiguration
+import com.exactpro.th2.read.db.core.QueryConfiguration
+import com.exactpro.th2.read.db.core.QueryParametersValues
+import com.fasterxml.jackson.annotation.JsonInclude
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+data class ExecuteBodyData(
+    val dataSource: DataSourceConfiguration,
+    val beforeQueries: List<QueryConfiguration> = emptyList(),
+    val query: QueryConfiguration,
+    val afterQueries: List<QueryConfiguration> = emptyList(),
+    val parameters: QueryParametersValues = emptyMap(),
+    val executionId: Long,
```

Comment:
I think the execution ID should be at the top of the list so users see it right away. You can either change its position here or you can use `@JsonPropertyOrder(["executionId"])` annotation